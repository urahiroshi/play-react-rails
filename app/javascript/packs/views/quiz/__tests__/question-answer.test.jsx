import React from 'react'
import { mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import QuestionAnswer from '../question-answer'

const createComponent = () => (
  <QuestionAnswer
    answer="hoge"
    onChange={() => {}}
    onEnterKey={() => {}}
  />
)

describe('#render', () => {
  it('should show snapshot', () => {
    const wrapper = mount(createComponent())
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
