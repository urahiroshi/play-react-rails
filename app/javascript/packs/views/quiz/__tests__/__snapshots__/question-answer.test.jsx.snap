// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render should show snapshot 1`] = `
<QuestionAnswer
  answer="hoge"
  onChange={[Function]}
  onEnterKey={[Function]}
>
  <div
    className="form-group"
  >
    <div
      className="col-sm-1 control-label quiz-header"
    >
      A.
    </div>
    <div
      className="col-sm-11"
    >
      <input
        className="form-control quiz-answer-input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="Input Answer !"
        type="text"
        value="hoge"
      />
    </div>
  </div>
</QuestionAnswer>
`;
